Doc/library/typing.rst

```diff
@@ -2112,7 +2114,7 @@ Other concrete types
       Python 2 is no longer supported, and most type checkers also no longer
       support type checking Python 2 code. Removal of the alias is not
       currently planned, but users are encouraged to use
-      :class:`str` instead of ``Text`` wherever possible.
+      :class:`str` instead of ``Text``.
```

Comment:
The only reason I can think of why it wouldn't be possible is that you're still supporting Python 2, and we don't need to care about that.